docs/tutorial/kymotracking.rst

```diff
@@ -0,0 +1,206 @@
+First, we need to get an image to perform the tracing on. Let's grab a kymograph from a Bluelake file::
+
+    file = lk.File('kymograph.h5')
+    _, kymo = file.kymos.popitem()
+
+We can extract the image data with one of the `_image` commands::
+
+    data = kymo.green_image
+
+Let's have a look at what it looks like::
+
+    plt.imshow(data, aspect="auto", vmax=5)
+
+.. image:: kymo_unscaled.png
+
+What we also see is that we can clearly see the bead edges in the kymograph. Let's only take a subsection of the
+kymograph, to make sure that we don't get any spurious line detections that come from the bead edges. We only use the
+70th to the 220th pixel.
+
+    >>> data = data[70:220, :]
+
+If we zoom in a bit, we can see that our traces in this image are about 3 pixels wide, so let's set this as our
+`line_width`.
+
+.. image:: kymo_zoom.png
+
+The peaks have an intensity of about 3-7 photon counts, whereas the background fluctuates around 0-2. Let's set our
+`pixel_threshold` to 3. We also see that sometimes, a particle momentarily disappears. To still connect these in a
+single trace, we want to allow for some gaps in the connection step. Let's use a `window` size of 6 in this test.
+Running the algorithm is easy::
+
+    traces = lk.track_greedy(data, line_width=3, pixel_threshold=3, window=6)
+
+The result of tracking is a list of kymograph traces::
+
+    >>> print(len(traces))  # the number of traces found in the kymo
+    745
+
+Sometimes, we can have very short spurious traces. To remove these from the list of detected traces we can use
+filter_lines. To omit all traces with fewer than 4 detected points, we can invoke::
+
+    >>> traces = lk.filter_lines(traces, 3)
+
+We can get the determined time and coordinate indices of these traces from the `time_idx` and `coordinate_idx`
+attributes. Let's plot the first trace::
+
+    plt.plot(traces[0].time_idx, traces[0].coordinate_idx)
+
+Plotting all the detected traces is also quite easy by iterating over the list of traces::
+
+    plt.imshow(data, aspect="auto", vmax=5)
+    for trace in traces:
+        plt.plot(trace.time_idx, trace.coordinate_idx)
+
+.. image:: kymotracked.png
+
+Fortunately, the signal to noise level in this kymograph is quite good. In practice, when the signal to noise is lower,
+one will have to resort to some fine tuning of the algorithm parameters over different regions of the kymograph to get
+an acceptable result.
+
+Note that the determined time and coordinates of the traces and coordinates are defined in pixels and therefore need to
+be converted to actual time and position using the delta time and pixel size of the kymograph. We can get the time
+between frames directly from the kymograph::
+
+    dt = kymo.delta_t
+
+And the pixel size as well::
+
+    pixel_to_um = kymo.pixelsize_um
+
+This allows conversion from pixels back to position.
+
+The second algorithm present is an algorithm that works purely on signal derivative information. It works by blurring
```

Comment:
This should get its own section title since the last one only mentions `greedy`.

```suggestion
Using the lines algorithm
-------------------------

The second algorithm present is an algorithm that works purely on signal derivative information. It works by blurring
```

Reply:
👍